charts/node/values.yaml

```diff
@@ -49,6 +49,7 @@ ingress:
 
 node:
   chain: "polkadot"
+  command: "polkadot"
```

Comment:
isn't the command always `polkadot`?

Reply:
It is for westend, kusama and polkadot but not for other substrate chains and parachains.